feature-libs/organization/administration/components/shared/current-organization-item.service.ts

```diff
@@ -31,7 +31,7 @@ export abstract class CurrentOrganizationItemService<T> {
    * The active item is loaded by the active `key$`.
    */
   readonly item$: Observable<T> = this.key$.pipe(
-    switchMap((code: string) => (code ? this.getItem(code) : of(null)))
+    switchMap((code: string) => this.getItem(code))
```

Comment:
The former code meant to prevent calling backend when there was no item code in the route. Are you sure, we like to remove it?

Reply:
removed